xpdtools/tools.py

```diff
@@ -581,3 +583,85 @@ def inner(x, *args, **kwargs):
         return func(*args, **kwargs)
 
     return inner
+
+
+def max_value(g, r):
+    """Returns largest value
```

Comment:
This is not so helpful. Bear in mind what the docstring is for.  It is for a user to understand what this function is for and how it should be used.  Also, the docstring shouldn't include the function name, which is effectively what your's does.

In other words, why do we (or rather a user) need this function (rather than just typing np.amax())?